commercetools-sdk-base/src/main/java/io/sphere/sdk/json/SphereJsonUtils.java

```diff
@@ -131,12 +136,11 @@ public static String prettyPrint(final JsonNode jsonNode) {
     }
 
     /**
-     *
      * Reads a UTF-8 JSON text file from the classpath of the current thread and transforms it into a Java object.
      *
-     * @param resourcePath the path to the resource. Example: If the file is located in "src/test/resources/foo/bar/product.json" then the path should be "foo/bar/product.json"
+     * @param resourcePath  the path to the resource. Example: If the file is located in "src/test/resources/foo/bar/product.json" then the path should be "foo/bar/product.json"
      * @param typeReference the full generic type information about the object to create
-     * @param <T> the type of the result
+     * @param <T>           the type of the result
```

Comment:
@lauraluiz you are right, it was IntelliJ who changes the documentation, it happened when I used the command to format the code. I will keep it as before :)